core/src/layers/capability_check.rs

```diff
@@ -191,7 +191,7 @@ impl<A: Access> LayeredAccess for CapabilityAccessor<A> {
         args: OpList,
     ) -> crate::Result<(RpList, Self::BlockingLister)> {
         let capability = self.info.full_capability();
-        if !capability.list_with_version && args.version() {
+        if !capability.list_with_version && args.versioned() {
```

Comment:
`list_with_version` should also be depreated by `list_with_versioned`.

Reply:
No problem I'll add that all also.